scripts/core/ui/components/generic-form/form-field.tsx

```diff
@@ -43,9 +46,13 @@ export function getFormFieldPreviewComponent(
     item: {readonly [key: string]: any},
     formFieldConfig: IFormField,
     options: { showAsPlainText?: boolean } = {},
-): JSX.Element {
+): JSX.Element | null {
     const Component = getFormFieldComponent(formFieldConfig.type);
 
+    if (Component == null) {
```

Comment:
Why would it be `null`?

Reply:
I was adding a hidden field, which wouldn't be rendered, but at the end I'm not using it

Reply:
I'm not getting it. How would a "hidden field" work here? From what I understand, all field types should have a corresponding component that would render the input field.

Reply:
it was like `<input type="hidden" value="...">`, was using it for `user_id`, but then I've moved that handling to the server

Reply:
I think `component_parameters` can be used for holding extra info that you tried to put inside a hidden field. Or if it doesn't work in some scenario, maybe it'd be worth adding "hidden field" input type. I'd drop this for now so we don't carry dead code that we would be afraid to remove in the future.